Detectors/MUON/MID/Raw/test/testRaw.cxx

```diff
@@ -137,6 +137,7 @@ BOOST_AUTO_TEST_CASE(ColumnDataConverter)
 
   std::vector<o2::mid::ROFRecord> rofs;
   std::vector<o2::mid::LocalBoardRO> outData;
+
```

Comment:
consider removing the inserted blank line

Reply:
Most likely due to merge conflicts, I will check.